app/views/staff/planned_disbursements/new.html.haml

```diff
@@ -6,5 +6,5 @@
       %h1.govuk-heading-xl
         = t("page_title.planned_disbursement.new")
 
-        = form_with model: @planned_disbursement, url: activity_planned_disbursements_path(@activity) do |f|
-          = render partial: "form", locals: { f: f }
+      = form_with model: @planned_disbursement, url: activity_planned_disbursements_path(@activity) do |f|
+        = render partial: "form", locals: { f: f }
```

Comment:
I wasn't sure what was going on with the indentation here, so I changed it back on my local and had a look at the interface to see the difference. Well spotted!

Reply:
Yeah - it was a good spot!